src/api/invitations.ts

```diff
@@ -68,6 +68,61 @@ export const createInvitations = async (invitationList: Array<object>) => {
     }
 }
 
+export const getEcosystemList = async () => {
+
+	const url = `${apiRoutes.Ecosystem.root}`
+
+	const token = await getFromLocalStorage(storageKeys.TOKEN)
+
+	const config = {
+			headers: {
+					'Content-Type': 'application/json',
+					Authorization: `Bearer ${token}`
+			}
+	}
+	const axiosPayload = {
+			url,
+			config
+	}
+
+	try {
+			return await axiosGet(axiosPayload);
+	}
+	catch (error) {
+			const err = error as Error
+			return err?.message
+	}
+}
+export const createEcoSystemInvitations = async (invitationList: Array<object>,ecosystemId: string) => {
+  	const orgId = await getFromLocalStorage(storageKeys.ORG_ID);
+	// const url = `${apiRoutes.organizations.root}/${orgId}${apiRoutes.organizations.invitations}`
```

Comment:
please let me know if this commented line is necessary

Reply:
Removed the commented code